apps/ledger-live-mobile/src/screens/CustomImage/Step2Preview.tsx

```diff
@@ -108,25 +106,6 @@ const Step2Preview = ({ navigation, route }: NavigationProps) => {
       [setProcessorPreviewImage],
     );
 
-  const handleRawResult: ImageProcessorProps["onRawResult"] = useCallback(
```

Comment:
keep this here instead of moving it to PreviewPostEdit, to have all the processing happening in this screen only

Reply:
Done